source/marsGameSound.js

```diff
@@ -20,15 +20,15 @@ this.initialize = function() {
     for ( var i = 0; i < soundSet.length; ++i ) {
         this.loadSound( soundSet[i] );
     }
-    this.future( 0 ).setUpSubtitles();
+    this.future( 0.01 ).setUpSubtitles();
```

Comment:
Okay, right, I remember that conversation.  It was basically that if your `application.initialize` is doing `future( 0 )` things that you need to happen before this, then you'll want to set this one up to run at `future( 0.01 )` so as to make sure that the `application.initialize` `future( 0 )` happens first.  I assume that is the case here?